### ISSUE TITLE
Ad an easier first level

### ISSUE THREAD

**Issue by Glitchy-Tozier** [pserwylo/beat-game#62]
The fist level is **really** difficult. To make the learning experience less frustrating to the user, I suggest adding an easier level that is less overwhelming.

Something basic, starting with only individual posts/buildings, then move to mutlipe posts/buildings, then use _only a few_ posts on buildings (like you do in the current level, but less obstacles)

**Comment by pserwylo** [comment]
Thanks, in v0.2.0 (currently in F-Droid) the first level is "Maintenance". I did that originally because it was extremely short, but I've since gotten a bit of feedback offline that is similar to yours, in that it is not a pleasant experience.

I think it is fair to say that "The Laundry Room" is the simplest level, so I've moved that to the first level for v0.3.0 (tagged but not yet released, though there is a debug build avaialble via GitHub releases if you're feeling adventurous). 

My main concern with using this as the first level originally is that it is too boring, and doesn't really show off the features of the game, but I've definitely come around to the idea that it **needs** to be a simple level first.
